.github/workflows/check-labels.yml

```diff
@@ -8,6 +8,9 @@ on:
       - labeled
       - unlabeled
       - edited
+concurrency:
+  group: check-labels-${{ github.event.number }}
```

Comment:
Could we use `${{ github.workflow }}-${{ github.ref }}` as well to cancel the in-progress runs of the current workflow? ([docs](https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow))

Reply:
Aaa, I see, thanks for the clarification 👍